api/Dockerfile

```diff
@@ -1,33 +1,29 @@
 # Remember: the context for this build is the root of the monorepo,
 # not the api folder
 
-# TODO: we only use bun as a dependency manager, but api is still executed with
-# node. We should eventually get rid of node
-FROM oven/bun:slim as base
+# We need Node to run prisma generate. See https://github.com/oven-sh/bun/issues/3827
+# TODO: switch to oven/bun-node:slim
```

Comment:
In any case shouldn't our eventual goal be to use `oven/bun:slim` and remove all dependency on Node?

Reply:
Oh that's my bad, I meant to write bun not bun-node